test/UrlHelperTest.php

```diff
@@ -48,7 +48,7 @@ public function testRaisesExceptionOnInvocationIfNoRouteProvidedAndResultIndicat
 
     public function testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided()
     {
-        $this->router->generateUri('foo', [])->willThrow(RouterException::class);
+        $this->router->generateUri('foo', [], [])->willThrow(RouterException::class);
```

Comment:
Do we need to change these existing tests? Since the argument is optional, we should be able to keep these as-is, shouldn't we?